services/tasks.py

```diff
@@ -95,7 +125,17 @@ def run(self, service_id, **kwargs):
             status = self.get_health(service.url, service.token)
             try:
                 result = status.json()
+
+                if (not service.up and result['up'] and service.last_up_at):
+                    downtime = timezone.now() - service.last_up_at
+                    fire_metric.apply_async(kwargs={
+                        "metric_name": 'services.downtime.sum',
```

Comment:
I think we should be firing metrics per service so that they can be individually monitored.
so the metric_name would be something like `services.downtime.<service.name>.sum